src/IIIFPresentation/API.Tests/Helpers/PaintedResourceXTest.cs

```diff
@@ -86,9 +86,9 @@ public void GetRequiredNumberOfCanvases_0_IfPaintedResourcesEmpty()
     public void GetRequiredNumberOfCanvases_Correct_IfAllPaintedResourcesNullCanvasOrder()
     {
         List<PaintedResource> paintedResources = [
-            new() { CanvasPainting = new CanvasPainting { CanvasId = "1" } },
-            new() { CanvasPainting = new CanvasPainting { CanvasId = "2" } },
-            new() { CanvasPainting = new CanvasPainting { CanvasId = "3" } },
+            new() { CanvasPainting = new CanvasPainting() },
```

Comment:
There are now no longer any tests for `CanvasPainting` records that have `CanvasId` - seems like we should still have some?